node_modules/oae-core/resetpassword/bundles/default.properties

```diff
@@ -0,0 +1,13 @@
+ACCOUNT_NOT_CREATED = Account not created.
```

Comment:
All of these seem to already be in `/shared/oae/bundles/ui/`, we don't need to re-specify them here.